src/prisma/schema.prisma

```diff
@@ -1939,3 +1941,49 @@ model AdditionalBlockQuota {
   blockCount Int
   createdAt  DateTime  @default(now())
 }
+
+model Form {
+  id         String      @id @default(uuid()) @db.Uuid
+  formFields FormField[]
+  proposal   Proposal[]
+}
+
+enum FormFieldType {
+  short_text
+  long_text
+  number
+  email
+  phone
+  url
+  select
+  multiselect
+  wallet
+  date
+  person
+  label
+}
+
+model FormField {
+  id            String           @id @default(uuid()) @db.Uuid
+  type          FormFieldType
+  index         Int              @default(-1)
+  name          String
+  description   String?
+  required      Boolean          @default(false)
+  private       Boolean          @default(false)
+  options       Json?
+  parentFieldId String?          @db.Uuid
+  formId        String           @db.Uuid
+  form          Form             @relation(fields: [formId], references: [id], onDelete: Cascade)
+  fieldValues   FormFieldValue[]
+}
+
+model FormFieldValue {
```

Comment:
I think we should call this ProposalFormAnswer - this will be consistent with rubric and actually those are answers related to form questions

Reply:
Updated